README.md

```diff
@@ -376,6 +376,27 @@ export default Ember.Route.extend({
 });
 ```
 
+### Think out of the Shoebox
+
+Shoebox gives you great capabilities, but using in in the real app is pretty rough. Have you ever thought that such kind of logic should be done behind the scenes? In a large codebase, defining `fastboot.isFastboot` conditionals can be a daunting task. Furthermore, it generates a lot of boilerplate code, which obscures the solution. Sooner or later coupling with `shoebox` will spread over all routes.
```

Comment:
@HarenBroog typo: `using in in the real app` should be `using **it** in the real app`?